lib/spreewald/development_steps.rb

```diff
@@ -1,7 +1,7 @@
 # coding: UTF-8
 
-# Marks scenario as pending
-Then /^it should work$/ do
+# Marks scenario as pending, optionally explained with a reason.
+Then /^it should work(?:.+?)?$/ do
```

Comment:
Without the outer lazy operator `?`, the reason would no longer be optional. Omitting the inner one results in a warning:

```
warning: nested repeat operator '+' and '?' was replaced with '*' in regular expression: /^it should work(?:.+)?$/`
```